library/src/main/java/com/devbrackets/android/exomedia/core/api/VideoViewApi.java

```diff
@@ -112,6 +112,10 @@
 
     void setTrack(ExoMedia.RendererType type, int trackIndex);
 
+    void setRendererTrackGroupIndex(ExoMedia.RendererType trackType, int trackGroupIndex);
```

Comment:
Yes. The original intent was to have it work for both the adaptive tracks as well as things like CC.